cockatrice/src/remoteclient.cpp

```diff
@@ -27,7 +27,7 @@ static const unsigned int protocolVersion = 14;
 
 RemoteClient::RemoteClient(QObject *parent)
     : AbstractClient(parent), timeRunning(0), lastDataReceived(0), messageInProgress(false), handshakeStarted(false),
-      usingWebSocket(false), messageLength(0)
+      usingWebSocket(false), messageLength(0), hashedPassword()
```

Comment:
It took me a moment to realize this was a string, maybe "" as default init?

Reply:
no, using the default initaliser for this is good